docs/src/main/sphinx/language/sql-support.md

```diff
@@ -95,6 +95,13 @@ connector to connector:
 - {doc}`/sql/drop-materialized-view`
 - {doc}`/sql/refresh-materialized-view`
 
+(sql-catalog-management)=
+
+### Catalog management
+
+- {doc}`/sql/create-catalog`
+- {doc}`/sql/drop-catalog`
```

Comment:
I guess there is no ALTER or SHOW CREATE

Reply:
Not yet